config/table-headers.js

```diff
@@ -831,3 +831,78 @@ export const STATE_NORMAN = {
   default:   'unknown',
   formatter: 'BadgeStateFormatter',
 };
+
+/**
```

Comment:
@richard-cox     If all the variable definitions are put into the product file, I think it will make the file difficult to maintain, I think it would be better to define a specific harvester-table-headers.js file to hold these variables, what do you think?  
I would like to hear your opinion!